services/frontend/src/components/ProjectInfoForm/ProjectInfoForm.js

```diff
@@ -0,0 +1,28 @@
+import React from "react";
+import {TextField} from "@material-ui/core";
+import "./ProjectInfoForm.scss";
+
```

Comment:
In CDHSH-87, the naming of the parameter follows "___Input". I think it should be refactored here to match.

Reply:
I changed this not have `Input` in CDHSH-89 since the properties are used to apply the errors to invalid fields.